guide/16-geoenrichment-guides/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<ol><li>If text descriptions or the figure inserted here has a reference source, please cite here.</li><li>"Feature Engineering is a key aspect of any datascience project", please change to "Data Science"; Also, is Feature Engineering the only use case/application that geoenrichment is to be applied? If not, would be good to list them as bullet points, and give a brief intro here.</li><li>"To understand how GeoEnrichment adds (demographic or other) value(s), let's imagine a retail giant is evaluating potential sites to open new stores (when the conditions for evaluation include)<s>. For evaluation they need to look at</s> competition, traffic, economic feasibility and market potential of different geographic areas."</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Pts 1 and 3 are fixed. I have reworded pt 2 to make more sense. See if it helps.

Reply:
<p>For pt1, this is an Esri internal image used by GeoEnrichment so I have not cited a source. Do we need to do that for internal images as well?</p>